service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDetailsHelper.java

```diff
@@ -107,6 +108,10 @@ public static boolean isInGatekeepingState(CaseDetails caseDetails) {
         return isInState(GATEKEEPING, caseDetails);
     }
 
+    public static boolean isInGatekeepingListingState(CaseDetails caseDetails) {
```

Comment:
don't really see the value in this over isInState

Reply:
Removed